hospital-locator/src/SearchBar/SearchBar.js

```diff
@@ -1,17 +1,29 @@
 import React, { useState } from "react";
 import "./SearchBar.css";
-import cityList from "../__mocks__/cities.json";
+import locationDb from "location-database";
 
-const SearchBar = () => {
+const SearchBar = ({ updateSearchedCity }) => {
   const [searchCity, updateSearchCity] = useState("");
 
   const setSearchCity = (e) => {
     updateSearchCity(e.target.value);
   };
 
+  const handleCityClick = (e, city) => {
+    // console.log(city);
+    updateSearchedCity(city);
+    updateSearchCity("");
```

Comment:
Should both of these exist? Not sure what line 15 does (my buest guess is that it resets the content in the search box).

_I'm new to the project so it might just be a lack of familiarity._

Reply:
`updateSearchedCity` resets the search bar, and `updateSearchCity` updates the `searchedCity` state variable of the `HomePage` component, which is passed as a prop to the `HospitalList` component. 

This translates into the following scenario: user clicks a city from the dropdown, the search bar is reset, and the hospital list is updated.

Reply:
Exactly right, @PlasmaIntec !
@vishaalmehrishi Great callout that the names are probably confusing. Any suggestions on better, more distinct variable names here?

Reply:
How about `updateSearchCity` => `updateSearchBarCity` and `updateSearchedCity` => `updateUserCity`, since the results of `updateSearchedCity` are used to determine the user location?

Reply:
Thanks for the feedback! I'm going to make these changes and merge and we can handle the LocationWidget in a separate PR.